src/open_inwoner/openklant/migrations/0011_openklantconfig_api_sends_email_confirmation.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.10 on 2024-03-21 13:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("openklant", "0010_openklantconfig_register_channel"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="openklantconfig",
+            name="api_sends_email_confirmation",
+            field=models.BooleanField(
+                default=False,
```

Comment:
Lets turn this around to make the usage via the admin simpler: "Stuur contactformulier e-mailbevestiging" / "send_email_confirmation" which is off by default.